index.html

```diff
@@ -35,12 +39,199 @@ <h1 class="title">
+                <li class="about__li">CSS</li>
+            </ul>
+            <img src="./images/pounds.png" alt="" class="pound__logo">
+          </div>
+
+          <div class="frameworks">
+            <h3 class="h3">Frameworks</h3>
+            <ul class="about__ul">
+                <li class="about__li">Bootstrap</li>
+                <li class="about__li">Ruby on rails</li>
+                <li class="about__li">RSpec</li>
+                <li class="about__li">Capibara</li>
+                <li class="about__li">Selenium</li>
+            </ul>
+            
+            <img src="./images/Rectangle 19.svg" alt="" class="square__logo">
+          </div>
+
+          <div class="skills">
+            <h3 class="h3">Skills</h3>
+            <ul class="about__ul">
+                <li class="about__li">Codekit</li>
+                <li class="about__li">Codepen</li>
+                <li class="about__li">GitHub</li>
+                <li class="about__li">GitLab</li>
+                <li class="about__li">Terminal</li>
+            </ul>
+            <img src="./images/idea.png" alt="" class="idea__logo">
+          </div>
+        </section>
+
+        <section class="contact">
+            <div class="corner__geometry">
+                <img src="./images/corner_geometry.svg">
+            </div>
+
+            <h2 class="h2">Get Started</h2>
+            <p class="contact__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+            <form action="" class="contact__form">
+                <input type="text" placeholder="your name" name="name" class="name">
+                <input type="email" placeholder="your email" name="email" class="email">
+                <input type="text" placeholder="Enter your message here..." name="message" id="" class="message">
+                <input type="button" value="Start Collaboration" class="buttons contact__button">
+            </form>
+
+        </section>
     </main>
+
+    <footer class="footer">
+        <button class="buttons footer__button">
+            <h4 class="footer__button__text">Get my resume</h4>  
+            <img class="footer__button__image" src="./images/download_Resume.svg" alt="download my resume">    
+        </button>
+       
+        <section class="socials">
+            <a href=""><img src="./images/stackoverflow.svg" alt="go to my stackoverflow" class="social"></a>
+            <a href=""><img src="./images/github.svg" alt="go to my github" class="social"></a>
+            <a href=""><img src="./images/twitter.svg" alt="go to my twitter" class="social"></a>
+            <a href=""><img src="./images/linkendin.svg" alt="go to my linkendin" class="social"></a>
+            <a href=""><img src="./images/m.svg" alt="go to my M" class="social"></a>
+            <a href=""><img src="./images/fingers.svg" alt="go to my M" class="social"></a>
+        </section>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). It would be better if you put your icons in the `ul` tag and then per icon use the `li` tag 😁 😁